rpcs3/Emu/Cell/Modules/cellFs.cpp

```diff
@@ -20,23 +20,23 @@ s32 cellFsOpen(vm::cptr<char> path, s32 flags, vm::ptr<u32> fd, vm::cptr<void> a
 
 s32 cellFsRead(u32 fd, vm::ptr<void> buf, u64 nbytes, vm::ptr<u64> nread)
 {
-	cellFs.trace("cellFsRead(fd=0x%x, buf=0x%x, nbytes=0x%llx, nread=0x%x)", fd, buf, nbytes, nread);
+	cellFs.trace("cellFsRead(fd=0x%x, buf=0x%x, nbytes=0x%llx, nread=0x%x) -> sys_fs_read()", fd, buf, nbytes, nread);
```

Comment:
I used to use this syntax when function do something they are not supposed to or this is unclear. Howerer, here is quite the opposite.